src/YMenuButton.h

```diff
@@ -169,6 +184,16 @@ class YMenuButton : public YSelectionWidget
      **/
     YMenuItem * findMenuItem( int index, YItemConstIterator begin, YItemConstIterator end );
 
+    /**
+     * Recursively looks for the first item in the tree of the menu items
+     * using depth first search.
```

Comment:
Please add an example explaining what "path" is. I assume it is something like `[ "File", "Export", "As PDF" ]`, right?

Reply:
Added more details to the description.